pod/perlstyle.pod

```diff
@@ -8,7 +8,7 @@ Each programmer will, of course, have his or her own preferences in
 regards to formatting, but there are some general guidelines that will
 make your programs easier to read, understand, and maintain.
 
-The most important thing is to use L<strict> and L<warnings> in all your
+The most important thing is to use L<strict> and L<warnings> in all you
```

Comment:
'your' was correct

Reply:
Yes, see comment 2

It was a technical error from my side.